staging/src/k8s.io/sample-controller/controller.go

```diff
@@ -289,7 +289,7 @@ func (c *Controller) syncHandler(key string) error {
 	if !metav1.IsControlledBy(deployment, foo) {
 		msg := fmt.Sprintf(MessageResourceExists, deployment.Name)
 		c.recorder.Event(foo, corev1.EventTypeWarning, ErrResourceExists, msg)
-		return fmt.Errorf(msg)
+		return fmt.Errorf("%s", msg)
```

Comment:
Would it make more sense to switch this to `fmt.Error(msg)` instead? (given formatting isn't required at all)

Reply:
`fmt.Error()` doesn't exist. We could use `errors.New()`, but there was already an import named `errors` so I went this way. I can alias the import if you want.

Reply:
Think it's okay like this :)